DQMOffline/Trigger/plugins/NoBPTXMonitor.cc

```diff
@@ -0,0 +1,403 @@
+#include "DQMOffline/Trigger/plugins/NoBPTXMonitor.h"
+
+// -----------------------------
+//  constructors and destructor
+// -----------------------------
+
+NoBPTXMonitor::NoBPTXMonitor( const edm::ParameterSet& iConfig ) : 
+  folderName_             ( iConfig.getParameter<std::string>("FolderName") )
+  , jetToken_             ( mayConsume<reco::CaloJetCollection>      (iConfig.getParameter<edm::InputTag>("jets")      ) )   
```

Comment:
hi @jalimena - this should be consume not mayConsume (similarly on the next line)